package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "store",
+  "version": "0.1.0",
+  "private": true,
+  "author": "dmaberlin1",
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.3.0",
+    "@fortawesome/free-regular-svg-icons": "^6.3.0",
+    "@fortawesome/free-solid-svg-icons": "^6.3.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
+    "@reduxjs/toolkit": "^1.9.3",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.16",
+    "@types/react": "^18.0.28",
+    "@types/react-dom": "^18.0.11",
+    "axios": "^1.3.4",
+    "react": "^18.2.0",
+    "react-burger-menu": "^3.0.9",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.0.5",
+    "react-router-dom": "^6.9.0",
+    "react-scripts": "5.0.1",
+    "sass": "^1.59.3",
+    "typescript": "^4.9.5",
+    "validator": "^13.9.0",
+    "web-vitals": "^2.1.4"
+  },
```

Comment:
lot of those dependencies are dev-dependencies
doing like that you will put testing libraries in prod bundle and you don't want that